Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -155,16 +155,25 @@ open class Player: BaseObject {
         }
     }
 
-    fileprivate func unbindPlaybackEvents() {
+    private func unbindPlaybackEvents() {
         for id in playbackEventsListenIds {
             stopListening(id)
         }
 
         playbackEventsListenIds.removeAll()
     }
+    
+    public static func register(plugins: [Plugin.Type]) {
+        if(!hasAlreadyRegisteredPlugins) {
```

Comment:
Let's follow the Swift convention: there's a space after the `if` and we don't use brackets.

Reply:
Done!